### ISSUE TITLE
[COGNITION]: Heading levels could be changed on HCA alerts for improved screen reader nesting

### ISSUE THREAD

**Issue by peterhill-oddball** [department-of-veterans-affairs/va.gov-team#1082]
Migrated from [#18404](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/18404)

## Issue

The headings for HCA alerts like an applicant not qualifying for VA healthcare or missing financial listings could be improved by changing the heading level four to a heading level two.

## Acceptance Criteria

As a screen reader user, I want to hear "You didn't qualify for VA healthcare based on your previous application, heading level two". Screen reader users often navigate by heading and having the right heading nesting improves context and importance of content on the page.

## Environment

* https://staging.va.gov/health-care/apply/application/introduction
* Test user 371 (or any other test user in the HCA test plan)

## Possible Fixes (optional)

If the issue is one that can be fixed easily by changing HTML, CSS, or JavaScript markup, include the snippet in the issue description. If it is a longer fix or you do not have a ready solution, remove this section.

```html
// Would recommend using font utility classes from font size utilities
// https://design.va.gov/utilities/font-size
<h2 class="usa-alert-heading">
  You didn’t qualify for VA health care based on your previous application
</h2>
```

## WCAG or Vendor Guidance (optional)

* [Heading levels should only increase by one](https://dequeuniversity.com/rules/axe/3.2/heading-order)

---
![Screen Shot 2019-05-03 at 2 23 55 PM](https://user-images.githubusercontent.com/934879/57161240-aca9ed80-6db0-11e9-90e7-a33735d6797f.png)

**Comment by 1Copenut** [comment]
@Justin-Pickett I was just working through the backlog and noticed this one hadn't ever been assigned. Please feel free to re-assign or holler at me for more context. Thank you.

**Comment by Justin-Pickett** [comment]
@mojaray2k can you take a look into this ticket. Let me know if you need my assistance with anything in getting the info you need.

**Comment by Justin-Pickett** [comment]
@mojaray2k let me know when we can move this ticket to validate and get it closed out.

**Comment by Justin-Pickett** [comment]
@mojaray2k and @1Copenut can this ticket be closed out??

**Comment by 1Copenut** [comment]
@Justin-Pickett I did a quick check and there's a bit of related work to do. The alert was updated to an H2, but the headings beneath are H4s and should be H3s. LMK if you'd prefer to close this one and open a new related ticket to do that work.

---
![Screen Shot 2019-11-07 at 11.19.04 AM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/17cf4ca1-74d1-4f33-bd49-b127b90d3f73)

**Comment by Justin-Pickett** [comment]
@1Copenut yes that would be a great idea to close this ticket out as Amen is no longer on our team. Create a new ticket for the what needs to be done and i can get it plugged in with our new team member to get it fixed. Thanks.

**Comment by 1Copenut** [comment]
@Justin-Pickett Closing this issue and opening a related one in a moment. Thanks for the heads up!
